### ISSUE TITLE
Report directory is moved instead of copied while using Jenkins plugin

### ISSUE THREAD

**Issue by nkns165** [SahaginOrg/sahagin-java#19]
When using Sahagin plugin for Jenkins, Report directory (sahagin-report) is moved from workspace to build directory.
It is inconvenient when we want to download whole report files or save them as artifacts.  In addition, console log confused us. It says '**copy** from remote dir PATH_TO_WORKSPACE/sahagin-report to PATH_TO_BUILD_DIR/sahagin-report'.

I think it is resolve by removing this line:

``` java
reportOutputDir.deleteRecursive();
```

https://github.com/SahaginOrg/sahagin-jenkins-plugin/blob/a2e7cdb0a85a4ad61771fcbd3ad00c538ee5056d/src/main/java/org/sahagin/jenkins/SahaginReportPublisher.java#L65

but there is a comment like:

``` java
// TODO should use rename instead of copy for efficiency
```

Is there any specific reason not saving the file in workspace?

**Comment by NozomiIto** [comment]
Maybe, I was afraid that if the Sahagin report output logic was not called for unexpected reason such as build error and previous run report remained in workspace, the previous report might be misunderstood as the report for the current test run.

But, now I think this is not serious problem, and I can remove line `reportOutputDir.deleteRecursive();` if necessary.

**Comment by NozomiIto** [comment]
[sahagin.hpi.zip](https://github.com/SahaginOrg/sahagin-java/files/551064/sahagin.hpi.zip)

I've just create hpi without reportOutputDir deleting logic.
Can you try this? If there is no problem, I'll merge this fix

**Comment by nkns165** [comment]
Thank you so much for your quick fix.
I have confirmed that we can save original report directory with the new hpi.
Could you please merge it?

For your concern about remaining old result, I think we can usually avoid it by setting reportOutputDir under `build` directory (in case of Gradle. For Maven, `target`) and `clean` before testing.

**Comment by NozomiIto** [comment]
Thanks. I have committed the fix.

> I think we can usually avoid it by setting reportOutputDir under build directory (in case of Gradle. For Maven, target) and clean before testing.

This seems good idea, thank you!

**Comment by NozomiIto** [comment]
Close since fixed by [0.10.1](https://github.com/SahaginOrg/sahagin-java/releases/tag/0.10.1).
